### ISSUE TITLE
better epytext in twisted.application and twisted.conch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1943]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)| @mwhudson reported|
|-|-|
|Trac ID|trac#1943|
|Type|enhancement|
|Created|2006-07-20 20:54:14Z|

I spend a fair amount of time looking at the result of running Twisted's docstrings through epytext while working on pydoctor.  Often, the results aren't that pretty, so I thought I'd be systematic about it.  application and conch came first in the alphabet :-)

As well as fixing the odd ropey literal block, changing a lot of C{}s into L{}s and using @param a whole lot more, I fixed a few typos that I saw on my way.

You can see the output at:

http://starship.python.net/crew/mwh/apidocs/twisted.application.html
http://starship.python.net/crew/mwh/apidocs/twisted.conch.html

(you need a recent pydoctor for some of the links to work)

Not all the new L{}s in t.c.interfaces work yet, I'm not quite sure how to fix it.  One way is demonstrated in the docstring of IConchUser.lookupChannel, but that clutters the text version of the docstring rather).

Attachments:

* [better-epytest-appconch.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3cf/3cf2e7b1319b7b0fb8c74ff82339dcc07fb860c5/316afcf7c311c890abaac4639850ba278713e1f4.diff) (47478 bytes) - added by mwh on 2006-07-20 20:54:53Z - round 1

<details><summary>Searchable metadata</summary>

```
trac-id__1943 1943
type__enhancement enhancement
reporter__mwh mwh
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1153428854000000 1153428854000000
changetime__1153658604000000 1153658604000000
version__None None
owner__ 
cc__mwh
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17656]) Apply mwh's epytext cleanup patch

Author: mwh
Reviewer: exarkun
Fixes [#1943](https://github.com/twisted/twisted/issues/1943)

This fixes various errors in the epytext markup in twisted.conch and twisted.application.
It also makes changes to correct markup to make it more useful and so that it renders
in a more appealing way.  Documentation which was not using epytext markup at all has
also been converted to use it.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
